src/backend/mod.rs

```diff
@@ -1,6 +1,7 @@
 //! Backends for the Calyx compiler.
 pub mod mlir;
 pub mod resources;
+pub mod sexp;
```

Comment:
Again, if this backend is feature gated then we should feature gate the definition as well